frontend/src/app/pagination.ts

```diff
@@ -31,6 +31,16 @@ export interface TimeRangePaginationParams extends URLSearchParams {
   range_end: string;
 }
 
+export const DEFAULT_TIME_RANGE_PARAMS = {
+  order_by: 'time',
+  ascending: 'true',
+  filter: '',
+  range_start: new Date().toLocaleString('en-GB'),
+  range_end: new Date(
+    new Date().setMonth(new Date().getMonth() + 1)
+  ).toLocaleString('en-GB')
```

Comment:
I'm still a bit perplexed by the need to use Great Britain English as the locale string. It looks like the difference is ordering of month/day/year vs day/month/year, as well as military time vs. AM/PM.

~~~plaintext
new Date().toLocaleString("en-US")
> '5/26/2024, 12:36:51 PM'
new Date().toLocaleString("en-GB")
> '26/05/2024, 12:36:53'
~~~

My feeling is the API should be defaulting to ISO style strings:

~~~plaintext
new Date().toISOString()
'2024-05-26T16:37:56.630Z'
~~~

This format is pretty standard and timezone independent (the Z indicates it's been converted to Greenwich Standard Time +0000). Without this, it seems we are relying on the timezone of the client and server both being in agreement. This is _generally_ true, but not universally, and it will lead to surprising errors when these are out of agreement.

I'm going to create a ticket that references this PR as motivation for moving date endpoints to standardizing on ISO format.